docs/conf.py

```diff
@@ -8,52 +8,39 @@
 import sys
 
 import autoapi
+from importlib.metadata import version
 
 # Define path to the code to be documented **relative to where conf.py (this file) is kept**
 sys.path.insert(0, os.path.abspath('../src/'))
 
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = 'lsstseries'
-copyright = '2023, Doug Branton, Neven Caplar'
-author = 'Doug Branton, Neven Caplar'
-release = 'v0.1'
+project = "lsstseries"
+copyright = "2023, Doug Branton"
+author = "Doug Branton"
```

Comment:
Looks like Neven's name was dropped on lines 20 and 21.

Reply:
Changed to use the new lincc mailing list